dbtmetabase/metabase.py

```diff
@@ -729,8 +729,9 @@ def increase_indent(self, flow=False, indentless=False):
                     creator_email = creator.get("email")
                     creator_name = creator.get("common_name")
 
-                # No spaces allowed in model names in dbt docs DAG / No duplicate model names
-                exposure_name = exposure_name.replace(" ", "_")
+                exposure_label = exposure_name
+                # Only lower case letters and underscores allowed in model names in dbt docs DAG / No duplicate model names
+                exposure_name = re.sub("[^a-z0-9]", "_", exposure_name.lower())
```

Comment:
How does this affect non-Latin characters? Are they definitely not allowed in dbt?

Reply:
Good point, they would be replaced.
I also checked the dbt code again, the warning message is actually slightly wrong/misleading. So I adapted the replacement to be equivalent to the check they're doing.